### DISCUSSION TITLE
Register custom inactive function

### DISCUSSION THREAD

**Discussion by erick-xanadu** [EnzymeAD/Enzyme#1858] [Q&A] [answered]
I have several functions that I would like to register as inactive functions. I was wondering if there's a way to do so in a similar way to registering `enzyme_function_like_free`? Or is the only way to do so is by adding the `enzyme_inactive` attribute in the function directly?

Thanks!

**Comment by wsmoses** [comment, marked as answer]
Indeed, you can also make a global variable or the attribute! The global needs to contain the name `__enzyme_inactivefn`.
